docs/install/import.rst

```diff
@@ -53,6 +53,14 @@ To import a shapefile containing your paths, use the command ``loadpaths``::
         --srid=2154 --comments-attribute IT_VTT IT_EQ IT_PEDEST \
         --encoding latin9 -i
 
+After importing a large quantity of paths, it is recommended to pre-generate the
+paths graph needed for the routing. This action is not mandatory, but will reduce the time needed for the first
+routing following the import.
+
+To pre-generate the graph, use the ``generate_pgr_network_topology`` command::
+
+    docker compose run --rm web ./manage.py generate_pgr_network_topology
```

Comment:
We should not only give the command in Docker installation, but rather with the classic installation without Docker.